internal/policy/policy.go

```diff
@@ -223,7 +223,10 @@ func (s *Service) updatePolicies(ctx context.Context, actor Actor, svc, commitMs
 			return true, errors.WithMessagef(err, "make policies directory '%s'", policiesDir)
 		}
 
-		policiesPath := path.Join(policiesDir, fmt.Sprintf("%s.json", svc))
+		policiesPath, err := securejoin.SecureJoin(policiesDir, fmt.Sprintf("%s.json", svc))
+		if err != nil {
+			return true, errors.WithMessage(err, "join policy path")
```

Comment:
Should we add the `policiesDir` to this error to understand what happened?

Reply:
Yes, more context is a good idea 👍